handler/action_news_about.js

```diff
@@ -0,0 +1,79 @@
+const request = require('request-promise-native');
+const urls = require('../lib/urls');
+const fragmentSender = require('../lib/fragmentSender');
+const { buttonPostback, listElement } = require('../lib/facebook');
+
+const news_about = (chat, payload) => {
+    console.log(`Dialogflow Payload:`, payload);
+    search_id(payload).then(id => {
+        console.log(id);
+        request({
+            uri: urls.reports,
+            json: true,
+            qs: id,
+         }).then(report => {
+            if (report.length === 0) {
+                chat.sendText(`Dazu habe ich leider keine Info...🤔`)
+            }
+            else if (report.length === 1) {
+                payload.type = 'report';
+                chat.sendText(report[0].headline);
+                fragmentSender(chat, report[0].next_fragments, payload, report[0].text, report[0].media);
+            } else {
+                const elements = [];
+                report.forEach(r => {
+                    elements.push(listElement(r.headline, r.text, buttonPostback(
+                        'Lesen 📰', 
+                        {
+                            action: 'report_start',
+                            report: r.id,
+                            type: 'report',
+                        })
+                    ));
+                })
+                chat.sendList(elements);
+            }
+         }).catch(error => {
+             console.log(error);
+         });

```

Comment:
Consider flattening the promise chain here by returning the result from request.
Also: The above catch will result in no response to the user which is confusing.